CONTRIBUTORS.md

```diff
@@ -308,6 +308,7 @@ Created as part of the release process.
 + Wilhelmina Drengwitz
 + Win Wang
 + Wolfram Arnold
++ Worker Pants (Pantsbuild GitHub Automation Bot)
```

Comment:
@thejcannon I think we may not want this line in here, right..?

Reply:
He or she or they worked very hard.

🤷‍♂️🤷‍♂️🤷‍♂️🤷‍♂️ does it matter?

Reply:
maybe not.. just seems weird to me having a non-person on there 🤷🏽